include/cmocka.h

```diff
@@ -59,7 +59,7 @@ int __stdcall IsDebuggerPresent();
 
 /* If __WORDSIZE is not set, try to figure it out and default to 32 bit. */
 #ifndef __WORDSIZE
-# if defined(__x86_64__) && !defined(__ILP32__)
+# if (defined(__x86_64__) || defined((_LP64)) && !defined(__ILP32__)
```

Comment:
There seems to be an extra parentheses: ... defined((...

Reply:
Could be also:
+# if (defined(__x86_64__) && !defined(__ILP32__)) || defined(_LP64)

instead of:
-# if (defined(__x86_64__) && !defined(__ILP32__)) || defined(__sparc_v9__) || defined(__sparcv9)
...which is bad and do not work for Solaris amd64

Solaris is using _LP32 fo 32b mem.model and _LP64 for 64b mem.model on bot platforms (amd64,sparcv9) where both platforms can also run _LP32 mem.model.